mern/server/interactor/QA_interactor.js

```diff
@@ -0,0 +1,37 @@
+import QA_DAO from "../DAO/QA_dao.js";
+//Interactor calling the QA DAO functions after properly checking the inputs
```

Comment:
You need to create an instance of QA_DAO since it is a class.

let qaDAO = new QA_DAO(); Then you can use it in the functions.